packages/components/src/components/text-input/_text-input.scss

```diff
@@ -156,7 +156,8 @@
     color: $disabled-02;
     background-color: $disabled-01;
     border-bottom: 1px solid transparent;
-
+    // Needed to fix disabled text in Safari #6673
+    -webkit-text-fill-color: currentcolor;
```

Comment:
Random, is casing significant here or should it be `currentColor`?

Reply:
Oh weird, I thought casing makes a difference but I guess it doesn't! Works as is but I'm going to fix it anyways for consistency 😉